pandas/core/sorting.py

```diff
@@ -273,7 +275,7 @@ class _KeyMapper:
     Ease my suffering. Map compressed group id -> key tuple
     """
 
-    def __init__(self, comp_ids, ngroups, levels, labels):
+    def __init__(self, comp_ids, ngroups: int, levels, labels) -> None:
```

Comment:
```suggestion
    def __init__(self, comp_ids, ngroups: int, levels, labels):
```

Reply:
@simonjayhawkins Do you want to unannotate every ```__init__```?

Like removing the ```-> None``` from every` `` _init__```?

Reply:
if you come across any cases of `__init__` returning `None`, you can remove the `None`.